Makefile

```diff
@@ -31,27 +31,43 @@ manager: generate fmt vet
 	go build -o bin/manager ./cmd/manager/main.go
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: install
-	$(KUSTOMIZE) build config/local_run | kubectl apply -n $(NAMESPACE) -f -
+run: install install_crd install_rbac
 	eval $$(scripts/dev/get_e2e_env_vars.py $(cleanup)); \
 	go run ./cmd/manager/main.go
 
 # Install CRDs into a cluster
-install: manifests kustomize
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+install: manifests helm install_crd
+	
+
+install_crd:
```

Comment:
small nit, the other make targets use `kebab-case`, we should make sure that our naming conventions are consistent!